codalab/apps/teams/forms.py

```diff
@@ -1,14 +1,48 @@
 from django import forms
+from django.forms.widgets import ClearableFileInput, Input, CheckboxInput
+from django.utils.html import escape, conditional_escape
+from django.utils.encoding import force_unicode
 from .models import Team, TeamMembership
 from tinymce.widgets import TinyMCE
+from django.utils.html import format_html
+from django.utils.encoding import force_text
+from django.utils.safestring import mark_safe
 import os
 
+class CustomImageField(ClearableFileInput):
```

Comment:
I tried uploading a JS file and got this error: "Upload a valid image. The file you uploaded was either not an image or a corrupted image."

So it seems fine.